authenticator.js

```diff
@@ -11,6 +11,7 @@ function AuthenticateGithub(opts) {
     debug: true,
     githubHost: config.githubHost,
     githubPathPrefix: '/api/v3',
+    githubOrg: config.githubOrg,
```

Comment:
https://github.com/npm/npme-auth-github/compare/master...nathan7:multiple-github-orgs this would do the trick